Parsers/ASimProcessEvent/Parsers/ASimProcessTerminateMicrosoftSysmon.yaml

```diff
@@ -34,6 +34,32 @@ ParserQuery: |
                 User:string
               ) 
               with (regex=@'<Data Name="(\w+)">{?([^<]*?)}?</Data>')
+            | extend TargetUsername = User
+            | extend TargetUsernameType = iff(isnotempty(TargetUsername),'Windows', '')
```

Comment:
for terminate process, this is not mandatory it seems. let's skip this mapping
![image](https://github.com/user-attachments/assets/4f968d90-b3d0-4b18-aed0-83b152749830)

Reply:
rolled back changes